src/DynamoCoreWpf/UI/Prompts/DynamoMessageBox.xaml

```diff
@@ -5,6 +5,7 @@
         xmlns:ui="clr-namespace:Dynamo.UI"
         xmlns:localui="clr-namespace:Dynamo.Wpf.UI.GuidedTour"
         xmlns:w="clr-namespace:System.Windows;assembly=PresentationCore"
+        xmlns:fa5="http://schemas.fontawesome.com/icons/"
```

Comment:
This line is causing build error I think @aparajit-pratap

Reply:
Thanks for pointing this out @QilongTang